resources/src/operation/mod.rs

```diff
@@ -0,0 +1,220 @@
+use amount::Amount;
+use asset::AssetIdentifier;
+use serde::{de, Deserialize, Deserializer};
+use offer::PriceRatio;
+mod create_account;
+use self::create_account::CreateAccountFields;
+
+/// Operations are objects that represent a desired change to the ledger: payments, offers to
+/// exchange currency, changes made to account options, etc. Operations are submitted to the
+/// Stellar network grouped in a Transaction.
+
+#[derive(Debug, Serialize)]
+pub struct Operation {
+    id: i64,
+    paging_token: String,
+    type_i: u32,
+    operation_detail: OperationDetail,
+}
+
+#[derive(Debug, Deserialize, Serialize)]
+pub enum OperationDetail {
+    /// Create Account operation represents a new account creation.
+    CreateAccount(CreateAccountFields),
```

Comment:
I don't like the `Fields` suffix on these. Wouldn't just `CreateAccount` be fine? Or possibly `CreateAccountOp`? It just feels like there's a better descriptor although I'm not sure what it might be.